File: java-api/src/main/java/com/webmvc/javaapi/repository/custom/MemberCustomRepository.java
package com.webmvc.javaapi.repository.custom;

import com.webmvc.javaapi.domain.Member;
import org.springframework.data.domain.Page;
import org.springframework.data.domain.Pageable;

public interface MemberCustomRepository {

    Page<Member> findAllWithJpql(Pageable pageable, String account, String name);

    Page<Member> findAllWithQuerydsl(Pageable pageable, String account, String name);
}

File: spring-web-kotlin/src/main/kotlin/spring/web/kotlin/domain/member/repository/MemberCustomRepository.kt
package spring.web.kotlin.domain.member.repository

import org.springframework.data.domain.Page
import org.springframework.data.domain.Pageable
import spring.web.kotlin.domain.member.Member

interface MemberCustomRepository {

    fun findAllWithJpql(pageable: Pageable, account: String?, name: String?): Page<Member>
}